src/gmmbin/Makefile

```diff
@@ -5,10 +5,11 @@ EXTRA_CXXFLAGS = -Wno-sign-compare
 include ../kaldi.mk
 
 BINFILES = gmm-init-mono gmm-est gmm-acc-stats-ali gmm-align \
-           gmm-decode-faster gmm-decode-simple gmm-align-compiled \
-           gmm-sum-accs gmm-est-regtree-fmllr gmm-acc-stats-twofeats \
+           gmm-decode-faster gmm-decode-simple gmm-decode-nbest gmm-align-compiled \
```

Comment:
Did you forget to add gmm-init-trans.cc and gmm-decode-nbest.cc?

Reply:
I have made only one change in this file (i.e adding gmm-init-biphone)
The other changes must be due to a conflict.